docs/edge.md

```diff
@@ -2,11 +2,13 @@
 
 This page explains how to use Edge Positioning module in your bot in order to enter into a trade only if the trade has a reasonable win rate and risk reward ratio, and consequently adjust your position size and stoploss.
 
-!!! Warning
+  !!! Warning
```

Comment:
Indenting these will cause rendering to break, as the tripple exclamation mark MUST be the first 3 letters to render correctly on readthedocs.

Applies to the below Note too, obviously.